getting-started/get-the-sdk.md

```diff
@@ -16,8 +16,9 @@ For iOS and Android projects, the recommended approach to integrating the SDK is
 The Experience Platform SDK for Android supports Android 4.0 \(API 14\) or later.
 {% endhint %}
 
-1. Open a previously created and configured **Mobile** property in Experience Platform Launch, and click on the **Environments** tab, and then click on the install package icon![](../.gitbook/assets/package.png).
-2. On the **Mobile Install Instructions** dialog box, select **Android**.
+1. Open a previously created and configured mobile property in Experience Platform Launch and click on the **Environments** tab
+2. In the row for the **Development** environment, click on the install package icon![](../.gitbook/assets/package.png).
```

Comment:
the way the syntax is listed for the image is confusing, like it's part of the step text. Can we put the syntax in parantheses, like this:
"...click on the install package icon (image syntax).